.codesandbox/ci.json

```diff
@@ -0,0 +1,3 @@
+{
+  "sandboxes": ["new", "vanilla"]
```

Comment:
This is fixed, first, you should merge the first PR and pull it there, and should be OK.